### PR TITLE
Add EE 11 Core Profile Cert Results

### PR DESCRIPTION
**Pull request by brideck**
For certification of the OL 24.0.0.11-beta with EE 11 Core Profile.

The streams here continue to be strange. I only intended to add two new files, but there's a lot more than that in this PR.

### CODE REVIEW

File: jakartaee/11/coreprofile/24.0.0.11-beta-Java17-TCKResults.adoc [hunk 1]
```diff
@@ -0,0 +1,387 @@
+:page-layout: certification
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of Jakarta EE Core Profile 11.
```
Comment (jhanders34) [inline jakartaee/11/coreprofile/24.0.0.11-beta-Java17-TCKResults.adoc]:
```suggestion
As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], the following is a summary of the TCK results for Jakarta EE Core Profile 11.
```

File: jakartaee/11/coreprofile/24.0.0.11-beta-Java17-TCKResults.adoc [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 13, Failures: 0, Errors: 0, Skipped: 0
+[INFO] 
+[INFO] 
+[INFO] --- maven-failsafe-plugin:3.2.5:verify (verify) @ core-profile-tck-runner ---
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: 01:23 min
```
Comment (jhanders34) [inline jakartaee/11/coreprofile/24.0.0.11-beta-Java17-TCKResults.adoc:86]: This time is the same for both 17 and 21.  Is that a happy coincidence?

File: jakartaee/11/coreprofile/24.0.0.11-beta-Java21-TCKResults.adoc [hunk 3]
```diff
@@ -0,0 +1,387 @@
+:page-layout: certification
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of Jakarta EE Core Profile 11.
```
Comment (jhanders34) [inline jakartaee/11/coreprofile/24.0.0.11-beta-Java21-TCKResults.adoc]:
```suggestion
As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], the following is a summary of the TCK results for Jakarta EE Core Profile 11.
```

**Review by jhanders34 — CHANGES_REQUESTED** [review]

**Review by jhanders34 — COMMENTED** [review]
A few small edits and a question.

File: jakartaee/11/coreprofile/24.0.0.11-beta-Java17-TCKResults.adoc (hunk 2 above)
Comment (brideck) [reply jakartaee/11/coreprofile/24.0.0.11-beta-Java17-TCKResults.adoc:86]: Yep, just a coincidence. I noticed it when I was creating the docs and thought someone might ask about that.

**Review by jhanders34 — DISMISSED** [review]

File: jakartaee/11/coreprofile/24.0.0.11-beta-Java17-TCKResults.adoc [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+IBM Semeru Runtime Certified Edition 17.0.12.1 (build 17.0.12+7)
+Eclipse OpenJ9 VM 17.0.12.1 (build openj9-0.46.1, JRE 17 Linux amd64-64-Bit Compressed References 20240716_775 (JIT enabled, AOT enabled)
+OpenJ9   - 4760d5d320
+OMR      - 840a9adba
+JCL      - 9882134b8c0 based on jdk-17.0.12+7)
+----
+
+* Summary of the information for the certification environment, operating system, cloud, ...:
++
+Apache Derby 10.14.2.0
```
Comment (tevans78) [inline jakartaee/11/coreprofile/24.0.0.11-beta-Java17-TCKResults.adoc]: Which bit of core profile needed Derby?

File: microprofile/7.0/24.0.0.9-beta-cl240920240801-0303-TCKResults.adoc [hunk 5]
```diff
@@ -0,0 +1,63 @@
+:page-layout: certification
+= MicroProfile 7.0 TCK Results Summary
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the MicroProfile TCK results for MicroProfile 7.0.
+
+== Open Liberty 24.0.0.9-beta-cl240920240801-0303 MicroProfile 7.0 Certification Summary
+
+MicroProfile 7.0 consists of 8 MicroProfile Specifications plus the Jakarta EE Core Profile 10.0 specfication.
```
Comment (tevans78) [inline microprofile/7.0/24.0.0.9-beta-cl240920240801-0303-TCKResults.adoc:8]: This is a MP 7.0 + EE 10 result page, not EE 11 core profile

**Review by ayoho — DISMISSED** [review]

File: jakartaee/11/coreprofile/24.0.0.11-beta-Java17-TCKResults.adoc (hunk 4 above)
Comment (brideck) [reply jakartaee/11/coreprofile/24.0.0.11-beta-Java17-TCKResults.adoc]: That's a good catch. Nothing in core does, that's vestigial copypasta from Platform/Web Profile docs.

**Review by ayoho — APPROVED** [review]

**Pull request merged**